packages/radio-button/src/presenters/stylesheet.js

```diff
@@ -0,0 +1,82 @@
+export default function stylesheet(props, themeData) {
+  const { disabled } = props;
+
+  const opacity = disabled ? themeData["component.disabled.opacity"] : "1.0";
+
+  return {
+    // this maps to .radio-button
+    radioButton: {
+      display: "inline-block",
+      padding: "0 10px"
+    },
+
+    radioButtonContainer: {
+      position: "relative",
+      display: "flex",
+      flexDirection: "row",
+      alignItems: "center"
+    },
+
+    // this maps to .radio-button__wrapper
+    radioButtonWrapper: {
+      opacity,
+      position: "relative",
+      color: "transparent",
+      border: `${themeData["checkbox.borderWidth"]} solid ${
+        themeData["checkbox.borderColor"]
+      }`,
+      backgroundColor: "transparent",
+      textAlign: "center",
+      fontSize: "25px",
+      lineHeight: "14px",
+      height: themeData["checkbox.height"],
+      width: themeData["checkbox.width"],
+      borderRadius: "50%"
+    },
+
+    // this maps to .radio-button__input
+    radioButtonInput: {
+      display: "inline-block",
+      opacity: "0",
+      position: "absolute",
+      width: "100%",
+      height: "100%",
+      top: "0",
+      left: "0",
+      margin: "0",
+      padding: "0",
+      zIndex: "1",
+
+      // hover
+      "&:hover + span.radio-button__wrapper": {
```

Comment:
i know we discussed this a few times, and sorry if i'm still confused, but i'm no quite sure i understand why we can't use `isPressed`, `hasHover`, and `hasFocus` from the `ControlBehavior` component instead of doing this with css.

Reply:
RadioButton is unique from checkbox in that if one button is checked (within a group), then the checked state for every other button changes. We need a way of pushing these changes back into React or else it gets confused. It's doable but it adds a lot of complexity and we decided to go with the simplest solution first.

Reply:
that makes sense for the `checked` state, but would we not be able to use props for the other states? not sure if it's better/worse to mix them like that, but just curious.

Reply:
I think it's better to push this off until we figure out how to integrate the checked status into React. Most of the styles require some sort of special case when the radio button is checked so I think only the hover handler could be implemented in React.

Reply:
this makes sense to do down the road especially for `checked`. in doing some testing i believe i was able to trigger the `isPressed`, `hasHover`, and `hasFocus` props, not sure if you would still encounter the issue using them alongside `checked` but if interested, let me know and i can show you what i did in testing.